app/views/styleguide/pages/_header.erb

```diff
@@ -1,4 +1,7 @@
 <h2>ToC</h2>
 <ul>
+  <li>
+    <%= link_to "Onboarding Guide Page", styleguide_getting_started_guide_path %>
+  </li>
 </ul>
 <br/>
```

Comment:
As a suggestion: you could add OnboardingTopBar component to its own style page too